src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -214,4 +229,107 @@ public static void deleteTargetPubSubTopic() {
       }
     }
   }
+
+  /**
+   * Create Spanner Instance with 1 database containing sample tables
+   * with queries provided in file spannerTestDataCreateTableQueriesFile and spannerTestDataInsertDataQueriesFile.
+   */
+  @Before(order = 1, value = "@SPANNER_TEST")
+  public static void createTempSpannerInstance() throws InterruptedException, ExecutionException {
+    if (firstSpannerTestFlag) {
```

Comment:
Is it guaranteed that this method won't be called concurrently? If not, you'll need some synchronization

Reply:
Yes won't be called concurrently.